modules/renter/proto/formcontract.go

```diff
@@ -74,6 +75,16 @@ func FormContract(params ContractParams, txnBuilder transactionBuilder, tpool tr
 			{Value: types.ZeroCurrency, UnlockHash: types.UnlockHash{}},
 		},
 	}
+	if build.VersionCmp(host.Version, "1.3.0") >= 0 {
```

Comment:
for testing reasons this is 1.3.0. We should bump it to v1.3.1 after confirming that the tests pass